config/locales/controllers/en.yml

```diff
@@ -101,3 +101,6 @@ en:
           one: You may reset your password %{count} more time.
           other: You may reset your password %{count} more times.
         user_not_found: We couldn't find an account with that email address or username. Please try again.
+    suspension_notice_html: Your account has been suspended until %{suspended_until}. You may not add or edit content until your suspension has been resolved. Please <a href=\"#{new_abuse_report_path}\">contact Abuse</a> for more information.
```

Comment:
Creating the link like this doesn't work:
![Screenshot_20240609_153234](https://github.com/otwcode/otwarchive/assets/27060268/2a6eb2f4-4bbe-4273-a9b3-0646a4162c15)

Please use the link_to helper:

```rb
flash[:error] = t("users.suspension_notice_html", suspended_until: localize(current_user.suspended_until), contact_abuse_link: view_context.link_to(t("users.contact_abuse"), new_abuse_report_path))
```

```suggestion
    suspension_notice_html: Your account has been suspended until %{suspended_until}. You may not add or edit content until your suspension has been resolved. Please %{contact_abuse_link} for more information.
    contact_abuse: contact Abuse
```

Same for ban_notice_html, and please remove the duplicate suspension_notice_html entry.

It would be great if you could also remove the following lines from i18n-tasks.yml because they will no longer be needed:
https://github.com/otwcode/otwarchive/blob/dd6ec55a93d549cef4e9a43269dc1cebb975dd91/config/i18n-tasks.yml#L126-L128

You can ignore the "should be" comments, the translation keys you chose are better. However, you could move the two `*_notice_html` keys to `users.status` to align with #4338.

Reply:
Thanks, I was wondering how to use the link_to function in a controller.